helper_test.go

```diff
@@ -1727,6 +1753,32 @@ func createTeamToken(t *testing.T, client *Client, tm *Team) (*TeamToken, func()
 	}
 }
 
+func createTeamTokenWithOptions(t *testing.T, client *Client, tm *Team) (*TeamToken, func()) {
```

Comment:
Similar comment here, would expect that this would take a `TeamTokenCreateOptions` and pass it along